bin/sumd

```diff
@@ -127,9 +127,27 @@ def load_configs(schedule):
     for task in tasks:
         schedule.add(task)
 
+def riemann_client():
+    riemann = None
```

Comment:
No need for this `riemann = None` - each condition of the if statement sets a `riemann` variable.